### PR TITLE
[ADD] product_catalog: New module to have a product catalog in main m…

### PR DESCRIPTION
**Pull request by rra-odoo**
…enu (appswitcher).
<b>Task</b>: https://www.odoo.com/web#id=21254&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

<b>Pad</b>: https://pad.odoo.com/p/r.zLkCsQUqFVf99uI5

### CODE REVIEW

File: addons/product_catalog/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
```
Comment (xmo-odoo) [inline addons/product_catalog/__openerp__.py]: Lint issue: Statement seems to have no effect

File: addons/product_catalog/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    'name': 'Products Catalog',
+    'category': 'Sales Management',
+    'depends': ['product'],
+    'description': """
+Manage all aspects of Products from Product Catalog Menu
+========================================================
+
+Product Catalog supports variants, different unit of measures and properties.
```
Comment (bst-odoo) [inline addons/product_catalog/__openerp__.py]: Change string into : It is the central place to manage your products, its variants, unit of measures and other properties.

File: addons/product_catalog/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Products Catalog',
+    'category': 'Sales Management',
+    'depends': ['product'],
+    'description': """
+Manage all aspects of Products from Product Catalog Menu
```
Comment (bst-odoo) [inline addons/product_catalog/__openerp__.py]: Change string into : Manage your products and services from one app.

**Comment by tac-odoo** [comment]
Moved to https://github.com/odoo/odoo/pull/11387

**Pull request closed without merging**